common/test/acceptance/tests/lms/test_lms_edxnotes.py

```diff
@@ -11,7 +11,6 @@
 from ...fixtures.edxnotes import EdxNotesFixture, Note, Range
 
 
-@skipUnless(os.environ.get("FEATURE_EDXNOTES"), "Requires Student Notes feature to be enabled")
```

Comment:
NOTE: We should always run these bok-choy tests to make sure that other changes doesn't affect the Student Notes feature.